### PR TITLE
Motion blur fixes

### PR DESCRIPTION
**Pull request by Vsevolod1983**
NO RUSH WITH MERGE, PLEASE!
PR contains non-approved Core.

TICKET:  RPRMAYA-2456

PURPOSE:
Motion blur calculaion is wrong

EFFECT OF CHNAGES:
1. We dont take into account FPS anymore
2. To make motion blur we use Frames N and N + 1 instead of N and N - 1
3. Linear and angular calculation fixes
4. Camera exposure should be within range [0, 1] and counts in frames now

### CODE REVIEW

**Comment by bsavery** [comment]
Are we ready to start reviewing?

**Comment by Vsevolod1983** [comment]
Core guys introduced new changes in API regarding motion blur, so we need to update core version here and make appropriate changes. So PR needs to be updated

**Comment by Vsevolod1983** [comment]
Also, Brian, We need to approve and merge Core 2.1.7 first in order to merge this PR

**Comment by Vsevolod1983** [comment]
MB algorithm itself was updated yesterday

**Review by DMotorygin — APPROVED** [review]

File: FireRender.Maya.Src/FireRenderGlobals.cpp [hunk 1]
```diff
@@ -402,10 +402,10 @@ MStatus FireRenderGlobals::initialize()
 	Attribute::cameraMotionBlur = nAttr.create("cameraMotionBlur", "cmb", MFnNumericData::kBoolean, 0, &status);
 	MAKE_INPUT(nAttr);
 
-	Attribute::motionBlurCameraExposure = nAttr.create("motionBlurCameraExposure", "mbce", MFnNumericData::kFloat, 0.1f, &status);
+	Attribute::motionBlurCameraExposure = nAttr.create("motionBlurCameraExposure", "mbce", MFnNumericData::kFloat, 0.5f, &status);
 	MAKE_INPUT(nAttr);
 	nAttr.setMin(0.0);
-	nAttr.setSoftMax(10.0);
+	nAttr.setMax(1.0);
```
Comment (bsavery) [inline FireRender.Maya.Src/FireRenderGlobals.cpp:408]: Does setting it > 1 break RPR?  If not maybe should be a soft max.

**Review by bsavery — APPROVED** [review]

**Pull request merged**